interface/forms/clinical_notes/table.sql

```diff
@@ -0,0 +1,20 @@
+--
+-- Table structure for table `form_clinical_notes`
+--
+
+CREATE TABLE IF NOT EXISTS `form_clinical_notes` (
+  `id` bigint(20) NOT NULL,
+  `date` DATE DEFAULT NULL,
+  `pid` bigint(20) DEFAULT NULL,
+  `encounter` varchar(255) DEFAULT NULL,
+  `user` varchar(255) DEFAULT NULL,
+  `groupname` varchar(255) DEFAULT NULL,
+  `authorized` tinyint(4) DEFAULT NULL,
+  `activity` tinyint(4) DEFAULT NULL,
+  `code` varchar(255) DEFAULT NULL,
+  `codetext` text,
+  `description` text,
+  `external_id` VARCHAR(30) DEFAULT NULL,
+  `clinical_notes_type` varchar(30) DEFAULT NULL
```

Comment:
If list_options.option_id is the foreign key then should probably have the same length (100) here.

Reply:
got it thanks